c7n/utils.py

```diff
@@ -645,3 +645,7 @@ def parse(cls, data):
             filters.append(d)
 
         return filters
+
+
+def c7n_prefix_cache_key(s):
```

Comment:
after conversation with @kapilt , can you please rename it to `get_annotation_prefix`?